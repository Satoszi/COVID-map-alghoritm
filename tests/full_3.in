10 10 2 20
full_3
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
++++++++++
