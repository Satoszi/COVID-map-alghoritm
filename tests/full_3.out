9 9
5 9
0 9
7 8
2 8
4 7
9 6
1 6
7 5
3 5
5 4
0 4
9 3
7 2
4 2
1 2
9 0
6 0
3 0
0 0
